.pre-commit-config.yaml

```
@@ -26,7 +26,7 @@ repos:
     rev: pylint-2.6.0
     hooks:
       - id: pylint
-        files: ^nbqa/
+        files: "^(nbqa|tests)/"
```

I think this should still work
```suggestion
        files: ^(nbqa|tests)/
```